server/controllers/cohort.js

```diff
@@ -48,12 +49,14 @@ const getCohortsMetaData = (req, resp) => {
     .select('_id name description favIds memberIds ownerIds')
     .sort({ createdAt: -1 })
     .exec()
-    .then(docs =>
-      docs
-        ? resp.status(200).send(responseWithCohorts(docs, currentUserId))
-        : resp.status(404).send({ message: 'No cohorts data found.' })
-    )
-    .catch(err =>
+    .then(docs => {
```

Comment:
why this change?

Reply:
I wanted to make it consistent across every controller.